services/diddoc_service.go

```diff
@@ -122,21 +122,41 @@ func (ds DIDDocService) MarshallContentStream(contentStream protoiface.MessageV1
 	return string(result), nil
 }
 
+func FindId (Id string, requestedId string) bool{	
+	if strings.Contains(Id, "#"){
+		Id = strings.Split(Id, "#")[1]
+	}
+	if Id == requestedId {
+		return true
+	} else {
+		return false
+	}
```

Comment:
```suggestion
	return id == requestedId
```
+ could you please rename `Id` to `id`?